src/analysis/retail/monk/mistweaver/CHANGELOG.tsx

```diff
@@ -6,6 +6,7 @@ import { SpellLink } from 'interface';
 
 
 export default [
+  change(date(2022, 10, 13), <>Cleaned up MW spells/talents files</>, Trevor),
   change(date(2022, 10, 9), <>Added Secret Infusion haste buff and fixed Uplifted Spirits</>, Trevor),
   change(date(2022, 10, 8), <>Added support for existing MW talents</>, Trevor),
```

Comment:
I think it's acceptable to clean up the pre-DF stuff in changelog when updating a spec.